reference/docs-conceptual/components/vscode/using-vscode.md

```diff
@@ -126,7 +126,8 @@ the version:
 
 ### Adding your own PowerShell paths to the session menu
 
-You can add other PowerShell executable paths to the session menu through a Visual Studio Code setting.
+You can add other PowerShell executable paths to the session menu through a
```

Comment:
Might be worth giving the name of the setting if we think it's reliable enough to document here

Reply:
added